src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -276,6 +277,30 @@ cdef class Buffer:
         hb_buffer_set_message_func(self._hb_buffer, msgcallback, <void*>callback, NULL)
 
 
+cdef class Blob:
+    cdef hb_blob_t* _hb_blob
+    cdef object _data
+
+    def __cinit__(self, bytes data):
```

Comment:
I'm wondering if it makes sense to allow `str` and `Path` here and use `from_file_path` in this case, or would that be too clever and confusing?

Reply:
Hm no, i think it's better, less confusing to keep the two construction modes separate

Reply:
OK.